content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -33,37 +33,38 @@ folder structure. By default it will create:
 
 ```text
 /cypress
```

Comment:
We had asked in slack if this list is subject to change recently. There's a ticket for it once we know the extend of the new info https://cypress-io.atlassian.net/browse/UDX-100

But for now, this is the most up to date, considering the extensions.